IPython/nbconvert/filters/highlight.py

```diff
@@ -113,7 +113,7 @@ def _pygments_highlight(source, output_formatter, language='ipython', metadata=N
 
     # If the cell uses a magic extension language,
     # use the magic language instead.
-    if language == 'ipython' \
+    if language in ('ipython', 'ipython3') \
```

Comment:
I assumed that it worked in python2, but was too lazy to test.  Let me test now.